### ISSUE TITLE
[Bug] Issue with Bark

### ISSUE THREAD

**Issue by mantrakp2004** [coqui-ai/TTS#2913]
### Describe the bug

Getting traceback error while running bark

### To Reproduce

https://colab.research.google.com/drive/1MyT8loM6dyvPQiUj9y5jRm-GSJrP_-iB?usp=sharing

### Expected behaviour

N/A

### Environment

```shell
{
    "CUDA": {
        "GPU": [
            "Tesla T4"
        ],
        "available": true,
        "version": "11.8"
    },
    "Packages": {
        "PyTorch_debug": false,
        "PyTorch_version": "2.0.1+cu118",
        "TTS": "0.16.5",
        "numpy": "1.22.0"
    },
    "System": {
        "OS": "Linux",
        "architecture": [
            "64bit",
            "ELF"
        ],
        "processor": "x86_64",
        "python": "3.10.12",
        "version": "#1 SMP Fri Jun 9 10:57:30 UTC 2023"
    }
}
```

**Comment by erogol** [comment]
can you post the error and how to get that?

**Comment by mtnmecca** [comment]
bark uses numpy 1.25.2 and TTS uses numpy 1.22.0.  Here is the error

tts 0.16.6 requires numpy==1.22.0; python_version <= "3.10", but you have numpy 1.25.2 which is incompatible.
numba 0.57.0 requires numpy<1.25,>=1.21, but you have numpy 1.25.2 which is incompatible.

**Comment by mtnmecca** [comment]
solved from issue #2273 
pip install "numba==0.56.4"
pip install "numpy==1.23.5"

Currently running python 3.10.9
